src/lib/viewers/controls/experiences/ExperiencesContext.ts

```diff
@@ -5,6 +5,6 @@ export type Context = {
     experiences: Experiences;
 };
 
-export default React.createContext<Context>({
-    experiences: {},
-});
+const ExperiencesContext = React.createContext<Context>({ experiences: {} });
```

Comment:
Why was this changed?

Reply:
Just didn't notice the change after coming back from trying other approaches here. I'll revert it.

Reply:
Done. Resolving.